<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor ACOES: Perfil usuario</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="bSalirSinGuardar">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>560</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir sin Guardar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bGuardarYSalir">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>560</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guardar y Salir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNombre">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>20</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="lApellidos">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>50</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Apellidos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lNif">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>NIF</string>
    </property>
   </widget>
   <widget class="QLabel" name="lDireccion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Direccion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNombre">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eApellidos">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>50</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eNif">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>80</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eDireccion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>110</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="eComentarios">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/Fotos/imagenes/Prueba1.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPoblacion">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Poblacion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ePoblacion">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>170</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eCodigoPostal">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>140</y>
      <width>381</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lCodigoPostal">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo Postal</string>
    </property>
   </widget>
   <widget class="QLabel" name="lEstado">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Pais</string>
    </property>
   </widget>
   <widget class="QLabel" name="lTelefono1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefono </string>
    </property>
   </widget>
   <widget class="QLabel" name="lTelefono2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>250</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Telefono 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCorreo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Correo</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaAlta">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Alta</string>
    </property>
   </widget>
   <widget class="QLabel" name="lSector">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>220</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Sector</string>
    </property>
   </widget>
   <widget class="QLabel" name="lCertificado">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Certificado</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaSalida">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de salida</string>
    </property>
   </widget>
   <widget class="QLabel" name="lRelacion">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>280</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Relacion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eEstado">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>220</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eCertificado">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>340</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTelefono1">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>250</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eCorreoElectronico">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eSector">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eRelacion">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eTelefono2">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaAlta">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaSalida">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>310</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lProvincia">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Provincia</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eProvincia">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lUsuario">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLabel" name="lRol">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>420</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Rol</string>
    </property>
   </widget>
   <widget class="QLabel" name="lFechaNacimiento">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>370</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Fecha de Nacimiento</string>
    </property>
   </widget>
   <widget class="QLabel" name="lClave">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QLabel" name="lPermiso">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Permiso</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="eUsuario">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eFechaNacimiento">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eClave">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="eRol">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>420</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="ePermiso">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>450</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../recursosQT.qrc"/>
 </resources>
 <connections/>
</ui>
